client/scala/src/main/scala/ai/verta/utils/URLUtils.scala

```diff
@@ -0,0 +1,10 @@
+package ai.verta.utils
+
+import java.net.URLEncoder
+
+object URLUtils {
+  /** Utility method for HTML form encoding
+   *  @param input input
+   */
+  def urlEncode(input: String): String = URLEncoder.encode(input, "UTF-8").replaceAll("\\+", "%20")
```

Comment:
Let's put this in the client as I think everything has access to it. Having generic `utils` folders tends to make them into kitchen sinks with everything.

Reply:
That makes sense. I'll fix that.

Reply:
Resolved. I put the URLUtils in client package now.